Map Fixes + goonfurniture

TobyThorne opened the pull request:
- Added more bushes;
- Added couches;
- Added monkey plant;
- Changed grass color;
- Added more windows to the undergroud floor of Exodus;
- Added couches to the underground brig;
- Added a couch to Private Cabin Five;
- Added a couch to hydroponics;
- Added a couch to Abandoned Detective's Office;
- Added couches to the engineering break room;
- Fixed broken Undergroud Main area;
- Fixed lattices under the walls of the underground floor;
- Fixed missing underground botanics disposal pipe;
- Fixed missing botanics cables;
- Rearranged kitchen equipment;
Added a public garden seed storage to the public garden;

![OnyxBay - NSS Exodus 2020-08-18 064358](https://user-images.githubusercontent.com/45202681/90459534-68686b00-e123-11ea-87ea-99b906cdc67e.png)

- [x] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.
- [x] Я ознакомился c [Guide to Contribute](https://github.com/ChaoticOnyx/OnyxBay/blob/dev/docs/contributing.md).

MrMagick2104 commented:
Спрайты баков, имхо, прям так себе выглядят. Наши ровно такие же, но у них есть нормальные тени и перспектива. В общую тёмную гамму станции намного лучше вписываются, опять же, имхо.
А такие - прямой даунгрейд спрайтов.

MrMagick2104 commented:
По поводу перспективы - у нас 90% предметов имеют перспективу, как у ватертанков до этого ПРа: ящики, кофейные автоматы, картонные коробки, ульи с пчёлами. Прямо на скрине видно это. Не вписывается же.

TobyThorne commented:
> Спрайты баков, имхо, прям так себе выглядят. Наши ровно такие же, но у них есть нормальные тени и перспектива лучше. В общую тёмную гамму станции намного лучше вписываются, опять же, имхо.
> А такие - прямой даунгрейд спрайтов.

Старые спрайты - легаси десятилетней давности. Тут с гунов новые спрайты, тупо апгрейд под адекватную перспективу. Всякие мелкие предметы ещё нормально смотрятся в любой перспективе, а большие объекты - такое. Учитывая, что всякие стулья-столы-кровати-ванночки уже давно в 3/4.

Epicus7 commented:
1. Ватертанки у нас лучше были.
2. Куллера у нас теперь типо два будет или чо?

Все остальное вроде норм.

ArmolitskiyCarpoeb commented:
не вписывается стилистика кустов

не говорю, что сам спрайт куста говно, просто если посмотреть на другие спрайты кустов и на этот новый с гунов, то становится понятно, что новый не вписывается в нашу станцию

TRUST ME IM SPRITER

TobyThorne commented:
- Fixed missing underground botanics disposal pipe;
- Fixed missing botanics cables;
- Rearranged kitchen equipment;
- Reverted 3/4 reagent tanks;
- Added a public garden seed storage to the public garden;

TapKo4eB reviewed (approved):
заебал фичи в фиксы пихать

TobyThorne commented:
Дизайнерские аппрувы вижу, девчачьи аппрувы вижу, сжать-и-встроить вижу.
